tests/wpt/mozilla/tests/css/body_bgcolor_attribute_change.html

```diff
@@ -0,0 +1,13 @@
+<!doctype html>
+<html>
+  <head>
+    <link rel="match" href="body_bgcolor_attribute_change_ref.html">
+  </head>
+  <body bgcolor="yellow" onclick="this.setAttribute('bgcolor', 'green');">
+    Click me, I should become green.
```

Comment:
nit: No need for `onclick`, since this should be automated (and it is, below). So let's remove the `onclick` attribute, and change the text to something like "My background should be green".